### PR TITLE
תרגום לגרמנית

### PR DESCRIPTION
**Pull request by BerndKeller**

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    <string name="gpsEnabledDesc">Empfangen von standortbasierten Alarmierungen in Echtzeit.</string>
+    <string name="maxDistance">Maximale entfernung</string>
+    <string name="maxDistanceDesc">Schieben den Regler, um die maximale Entfernung von jeder Stadt auszuwählen, die eine Raketenwarnung empfängt.</string>
+    <string name="kilometer">km</string>
+    <string name="nearbyCities">in der nähe liegende Städte</string>
+    <string name="noLocation">Dein Standort konnte nicht ermittelt werden.</string>
+    <string name="noNearbyCities">Keine in der nähe liegenden Stäte, ändere die maximale Distance..</string>
+
+    <!-- GPS Frequency -->
+    <string name="gpsFrequency">Update interval</string>
```
Comment (eladnava) [inline app/src/main/res/values-de/strings.xml]: Translation missing?

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    <string name="minutes">minuten</string>
+
+    <!-- GPS Notification -->
+    <string name="enableGPSDesc">Bitte Aktiviere dein GPS Um standortbezogene Alarme zu erhalten.</string>
+    <string name="enable">Einschalten</string>
+
+    <!-- Alert Share -->
+    <string name="shareAlert">Teile die Meldung</string>
+    <string name="alertSoundedAt">Ein code-red-Alarm hörte sich aus A code red alert sounded off in</string>
+    <string name="atTime">at</string>
```
Comment (eladnava) [inline app/src/main/res/values-de/strings.xml]: Translation missing?

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    <string name="loading">Lade..</string>
+    <string name="testingGCM">Push Gateway wird Getestet... (1/2)</string>
+    <string name="testingPushy">Push Gateway wird Getestet... (2/2)</string>
+    <string name="gcmTestFailed">Der Selbsttest scheiterte teilweise. Es wurde keine Warnung vom Google-Push-Gateway empfangen, aber eine Warnung wurde erfolgreich über das sekundäre Push-Gateway empfangen. Bitte versuchen Sie es später noch einmal.</string>
+    <string name="pushyTestFailed">Der Selbsttest scheiterte teilweise. Es wurde kein Alarm vom sekundären Push-Gateway empfangen, aber eine Warnung wurde erfolgreich über das Google-Push-Gateway empfangen. Bitte versuchen Sie es später noch einmal.</string>
+    <string name="debugInfo">Zusätzliche Informationen: </string>
+    <string name="testSuccessful">Der Test wurde erfolgreich abgeschlossen!</string>
+    <string name="alerts">Alarmierungen</string>
+    <string name="advancedAlerts">Erweiterte Alarmierung</string>
+    <string name="integrations">Integration</string>
```
Comment (eladnava) [inline app/src/main/res/values-de/strings.xml]: Translation missing?

File: app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+    <!-- Volume -->
+    <string name="volume">Lautstärke</string>
+    <string name="volumeDesc">Stelle die Lautstärke der Alarmierung ein.</string>
+
+    <!-- Secondary Volume -->
+    <string name="secondaryVolume">Lautstärke</string>
+    <string name="secondaryVolumeDesc">Slide to select the desired volume for secondary rocket alerts.</string>
+
+    <!-- Secondary Cities -->
+    <string name="selectedSecondaryCities">Secondary cities</string>
```
Comment (eladnava) [inline app/src/main/res/values-de/strings.xml]: Translation missing?

File: app/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+    <string name="selectedSecondaryCitiesDesc">Select cities to receive secondary alerts about.</string>
```
Comment (eladnava) [inline app/src/main/res/values-de/strings.xml]: Translation missing?

File: app/src/main/res/values-de/strings.xml [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+    <string name="updatePositive">Update</string>
+    <string name="notNow">nicht jetzt</string>
+
+    <!-- Volume -->
+    <string name="volume">Lautstärke</string>
+    <string name="volumeDesc">Stelle die Lautstärke der Alarmierung ein.</string>
+
+    <!-- Secondary Volume -->
+    <string name="secondaryVolume">Lautstärke</string>
+    <string name="secondaryVolumeDesc">Slide to select the desired volume for secondary rocket alerts.</string>
```
Comment (eladnava) [inline app/src/main/res/values-de/strings.xml]: Translation missing?

File: app/src/main/res/values-de/strings.xml [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+    <!-- Secondary Sound -->
+    <string name="secondarySoundSelection">Sekundär ton</string>
+    <string name="secondarySoundSelectionDesc">Wähle einen Ton für sekundäre Raketenwarnungen aus.</string>
+
+    <!-- Secondary Vibration Mode -->
+    <string name="secondaryVibrate">Aktiviere die vibration</string>
+    <string name="secondaryVibrateDesc">Sekundäre Raketenwarnungen lassen das Handy vibrieren.</string>
+66
+    <!-- Secondary Silent Mode Override -->
+    <string name="secondarySilentOverride">Override silent mode</string>
```
Comment (eladnava) [inline app/src/main/res/values-de/strings.xml]: Translation missing?

File: app/src/main/res/values-de/strings.xml [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+    <string name="shareDesc">Teile die App</string>
+    <string name="shareMessage">Ich verwende Red Alert für Android, um Echtzeit-Raketenwarnungen zu erhalten - https://redalert.me</string>
+    <string name="vibrate">Schalte die vibration ein</string>
+    <string name="vibrateDesc">Raketen Alarmierungen lassen Dein Handy vibrieren.</string>
+    <string name="sentVia">Gesendet mit</string>
+    <string name="problemDesc">Problem Beschreibung: </string>
+    <string name="noSound">Kein Ton</string>
+    <string name="search">Enter search keywords...</string>
+    <string name="allString">Alle</string>
+    <string name="noneString">None</string>
```
Comment (eladnava) [inline app/src/main/res/values-de/strings.xml]: Translation missing?

File: app/src/main/res/values-de/strings.xml [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+    <string name="sounds">Töne</string>
+    <string name="share">Teile</string>
+    <string name="shareDesc">Teile die App</string>
+    <string name="shareMessage">Ich verwende Red Alert für Android, um Echtzeit-Raketenwarnungen zu erhalten - https://redalert.me</string>
+    <string name="vibrate">Schalte die vibration ein</string>
+    <string name="vibrateDesc">Raketen Alarmierungen lassen Dein Handy vibrieren.</string>
+    <string name="sentVia">Gesendet mit</string>
+    <string name="problemDesc">Problem Beschreibung: </string>
+    <string name="noSound">Kein Ton</string>
+    <string name="search">Enter search keywords...</string>
```
Comment (eladnava) [inline app/src/main/res/values-de/strings.xml]: Translation missing?

File: app/src/main/res/values-de/strings.xml [hunk 10]
```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="appName">RedAlert</string>
+    <string name="signing_up">Signing up for push notifications...</string>
+    <string name="okay">OK</string>
+    <string name="error">Error</string>
```
Comment (eladnava) [inline app/src/main/res/values-de/strings.xml]: Translation missing?

File: app/src/main/res/values-de/strings.xml [hunk 11]
```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="appName">RedAlert</string>
+    <string name="signing_up">Signing up for push notifications...</string>
```
Comment (eladnava) [inline app/src/main/res/values-de/strings.xml]: Translation missing?

**Comment by BerndKeller** [comment]
German is sometimes a stupid language :/
https://www.youtube.com/watch?v=sbUuMW-vIY8

File: app/src/main/res/values-de/strings.xml (hunk 3 above)
Comment (BerndKeller) [inline app/src/main/res/values-de/strings.xml]: No

File: app/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (BerndKeller) [inline app/src/main/res/values-de/strings.xml]: yes

File: app/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (BerndKeller) [inline app/src/main/res/values-de/strings.xml]: no
A fusion of German <+> English  = Denglish; :P

File: app/src/main/res/values-de/strings.xml (hunk 5 above)
Comment (BerndKeller) [inline app/src/main/res/values-de/strings.xml]: yes

File: app/src/main/res/values-de/strings.xml (hunk 4 above)
Comment (BerndKeller) [inline app/src/main/res/values-de/strings.xml]: yes

File: app/src/main/res/values-de/strings.xml (hunk 6 above)
Comment (BerndKeller) [inline app/src/main/res/values-de/strings.xml]: yes

File: app/src/main/res/values-de/strings.xml (hunk 7 above)
Comment (BerndKeller) [inline app/src/main/res/values-de/strings.xml]: yes

File: app/src/main/res/values-de/strings.xml (hunk 8 above)
Comment (BerndKeller) [inline app/src/main/res/values-de/strings.xml]: yes

File: app/src/main/res/values-de/strings.xml (hunk 9 above)
Comment (BerndKeller) [inline app/src/main/res/values-de/strings.xml]: yes

File: app/src/main/res/values-de/strings.xml (hunk 10 above)
Comment (BerndKeller) [inline app/src/main/res/values-de/strings.xml]: No
_________________________
error = error  👍 
or
error = Fehler   👎 not so great , Even if the correct translation is

File: app/src/main/res/values-de/strings.xml (hunk 11 above)
Comment (BerndKeller) [inline app/src/main/res/values-de/strings.xml]: yes

**Pull request closed without merging**